### PR TITLE
remove moment.js

### PR DESCRIPTION
**Pull request by Unruly-Coder**
This small update saves hundreds of kb :)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -77,7 +79,12 @@ function getInfo (cpr) {
 // list all valid cpr numbers for a date, using modulo-11 and century check
 function validForDate (date) {
   var year = date.getFullYear()
-  var dateString = moment(date).format('DDMMYY')
+
+  var formattedDate = ('0' + date.getDate()).slice(-2);
+  var formattedMonth = ('0' + (date.getMonth() + 1)).slice(-2);
+  var formattedYear = date.getFullYear().toString().substr(2,2);
```
Comment (mathiasvr) [inline index.js]: Is there a reason for using `substr` and not `slice` here?

**Review by mathiasvr — COMMENTED** [review]
Thanks,
Could you run `npx standard --fix` for style?

File: index.js (hunk 1 above)
Comment (Unruly-Coder) [reply index.js]: no reason :) a late hour + copy/paste

**Comment by mathiasvr** [comment]
Thanks for the quick updates! 👍 

I'll merge and release as v2.1.0.

**Pull request merged**